src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -152,13 +152,13 @@ static uint32_t _window_guest_list_last_find_groups_tick;
 static uint32_t _window_guest_list_last_find_groups_selected_view;
 static uint32_t _window_guest_list_last_find_groups_wait;
 
-static int32_t _window_guest_list_highlighted_index; // 0x00F1EE10
-static int32_t _window_guest_list_selected_tab;      // 0x00F1EE12
-static int32_t _window_guest_list_selected_filter;   // 0x00F1EE06
-static int32_t _window_guest_list_selected_page;     // 0x00F1EE07
-static uint32_t _window_guest_list_selected_view;    // 0x00F1EE13
-static uint16_t _window_guest_list_num_pages;        // 0x00F1EE08
-static int32_t _window_guest_list_num_groups;        // 0x00F1AF22
+static uint32_t _window_guest_list_highlighted_index; // 0x00F1EE10
+static int32_t _window_guest_list_selected_tab;       // 0x00F1EE12
+static int32_t _window_guest_list_selected_filter;    // 0x00F1EE06
+static int32_t _window_guest_list_selected_page;      // 0x00F1EE07
+static uint32_t _window_guest_list_selected_view;     // 0x00F1EE13
+static uint16_t _window_guest_list_num_pages;         // 0x00F1EE08
+static uint32_t _window_guest_list_num_groups;        // 0x00F1AF22
```

Comment:
Are these comments still relevant?

Reply:
Personally, I prefer keeping them. But I'm not too fussed either way.

Reply:
Alright, then let's keep'em.